tests/example_workflows/pyxaid2/example_1_Si/step3/run_step3.py

```diff
@@ -108,25 +109,23 @@
 
 if setup==1:
     # ===  Basis of KS orbitals ====
-    params["norbitals"] = 12  # 6 pairs of KS orbitals, 12 spin-orbitals
-    params["active_space"] = range(2,6)+range(8,12)
+    params["norbitals"] = 44  # 22 pairs of KS orbitals, 44 spin-orbitals
```

Comment:
this would break my examples, if you want to add this case - add it as another setup